pyaedt/modules/PostProcessor.py

```diff
@@ -1167,6 +1178,17 @@ def export_report_to_file(self, output_dir, plot_name, extension, unique_file=Fa
                 * (Ansoft Report Data Files) .rdat
         unique_file : bool
             If set to True, generates unique file in output_dit
+        uniform : bool, optional
+            Whether if export uniform points to file or not.
+            Default is ``False``.
+        start : str, optional
+            If ``uniform`` is set then this is the start range with units for the sweep.
+        end : str, optional
+            If ``uniform`` is set then this is the end range with units for the sweep.
+        step : str, optional
+            If ``uniform`` is set then this is the step range with units for the sweep.
+        use_trace_number_format : bool, optional
+            Whether to use trace number formats or not. Default is ``False``.
```

Comment:
```suggestion
        uniform : bool, optional
            Whether the export uniform points to the file. The
            default is ``False``.
        start : str, optional
            Start range with units for the sweep if the ``uniform`` parameter is set to ``True``..
        end : str, optional
            End range with units for the sweep if the ``uniform`` parameter is set to ``True``.
        step : str, optional
            Step range with units for the sweep if the ``uniform`` parameter is set to ``True``..
        use_trace_number_format : bool, optional
            Whether to use trace number formats. The default is ``False``.
```
Some of the already existing parameter descriptions should be cleaned up also to match our standards.